### DISCUSSION TITLE
💡 Ideas / Suggestions

### DISCUSSION THREAD

**Discussion by AzaWoodyy** [FormationWebExpertBusiness/GestionStocksAlpes#143] [Ideas]
💡 **Ideas / Suggestions**

**Comment by xenoword** [comment]
On retire les quantitées ou on adapte la base pour s'en servir ?

Actuellement la quantité c'est le count d'Item donc on ne peut pas gérer avec des m pour le cable ou autre

solution :
- retirer l'uniter et gerer uniquement avec le count
- ajouter un champ quantité par défaut à 1

**Reply by AzaWoodyy** [reply]
Vaut mieux retirer les unités pour moi, s'il ils veulent la taille d'un rouleau par exemple ils le mettent en commentaire
